benchmark/zlib/pipe.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common.js');
+const fs = require('fs');
+const zlib = require('zlib');
+
+const bench = common.createBenchmark(main, {
+  inputLen: [1024],
+  duration: [5],
+  type: ['string', 'buffer']
+});
```

Comment:
These options have to be reflected in the benchmark test as well (`parallel/test-benchmark-zlib.js`).

Reply:
@BridgeAR they are no? See line 14.

Reply:
As far as I can tell, `inputLen` and `duration` have to be added to the test.

Reply:
Oh, true.

Reply:
Actually `inputLen` and `duration` are used in the test.

Reply:
@lpinca Can you clarify? At the very least we should include duration so that the test doesn't run it for 5s but rather like 1ms or similar. I don't think we necessarily need to include `inputLen` but we could too.

Reply:
Right now we just have:

```js
runBenchmark('zlib',
             [
               'method=deflate',
               'n=1',
               'options=true',
               'type=Deflate'
             ]);

Reply:
It seems this benchmark is like net benchmarks where throughput is calculated after x sec. I don't think this will be run in CI. Or I am not understanding the issue :)

Reply:
Right but we have tests that run these benchmarks to confirm that they work and also to confirm that we didn't break some functionality. See `test/parallel/test-benchmark-zlib.js`.

Reply:
I see now, I didn't read the original comment carefully.

Reply:
Done!